packages/docusaurus/src/client/exports/ComponentCreator.tsx

```diff
@@ -21,7 +24,8 @@ function ComponentCreator(path: string): ReturnType<typeof Loadable> {
     });
   }
 
-  const chunkNames = routesChunkNames[path];
+  const chunkNamesKey = `${path}-${hash}`;
```

Comment:
I prefered reconstructing the chunkNamesKey with the same logic on client because chunkNamesKey already contains the path so this would look more messy if user inspects .docusaurus folder, and would also take more place